.github/skills/floci-dev-assistant/evals/iteration-1/eval-1-translate-service/with_skill/run-1/outputs/response.md

```
@@ -0,0 +1,517 @@
+  --debug 2>&1 | grep "X-Amz-Target"
+# → X-Amz-Target: AWSShineFrontendService_20170701.TranslateText
+```
+
+Floci では **JSON 1.1 サービスのリファレンス実装は `services/ssm/` や `services/textract/`** です。
+Translate は状態を持たない（テキスト変換はステートレス）ため、Textract のパターンが最も近いです。
+
+---
+
+## Step 2: ディレクトリ構成
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)